zuul.d/jobs.yaml

```diff
@@ -297,6 +297,16 @@
       ansible_test_command: network-integration
       ansible_test_integration_targets: "eos_.*"
 
+- job:
+    name: ansible-test-units-eos
+    parent: ansible-test-units-base
+    required-projects:
+      - name: github.com/ansible-collections/arista.eos
+    timeout: 3600
```

Comment:
we skip this, as we set it in the parent. We can remove it in the next PR, same for openvswitch